src/pages/components/SecondaryBar.qml

```diff
@@ -85,7 +85,7 @@ Item {
         Browser.IconButton {
             width: midIconWidth
             icon.source: "image://theme/icon-m-file-download-as-pdf"
-            active: webView.contentItem && webView.contentItem.active && !webView.loading
+            active: webView.contentItem && webView.contentItem.active && !webView.loading && !DownloadManager.pdfPrinting
```

Comment:
Let's start with this one. After this we can update this to use the Notice component so that user can also follow what's happening.